File: 2018/src/main/java/nl/remcoder/adventofcode/day1/Part1.java
package nl.remcoder.adventofcode.day1;

import java.nio.file.Files;
import java.nio.file.Paths;

public class Part1 {
    public static void main(String[] args) throws Exception {
        System.out.println(Files.lines(Paths.get(ClassLoader.getSystemResource("day1/input").toURI()))
             .mapToInt(Integer::parseInt)
             .sum());
    }
}

File: 2018/src/main/java/nl/remcoder/adventofcode/day1/Part1.kt
package nl.remcoder.adventofcode.day1

import java.nio.file.Files
import java.nio.file.Paths

fun main (args : Array<String>) {
    println(Files.lines(Paths.get(ClassLoader.getSystemResource("day1/input").toURI()))
            .mapToInt(Integer::parseInt)
            .sum())
}